README.md

```diff
@@ -48,3 +48,34 @@ go run cmd/codegen/main.go -t mksdk
 cd ../generated/pango
 PANOS_HOSTNAME='***' PANOS_USERNAME='***' PANOS_PASSWORD='***' go run example/main.go
 ```
+
+## Acceptance testing
+The acceptance test suite creates real resources in a configured instance. We need to set the following environment variables in order to run an acceptance test against the PANOS provider.
+
+```sh
+# set up the Terraform plugin testing framework in acceptance testing mode
+# explicitly to allow the creation of real resources using a provider
+# See: https://developer.hashicorp.com/terraform/plugin/testing/acceptance-tests#requirements-and-recommendations
+export TF_ACC=1
+
+# Provider credentials
+export PANOS_HOSTNAME=...
+
+# if we have self-signed certificate for a testing instance
+export PANOS_SKIP_VERIFY_CERTIFICATE=true
+
+# We use either API key or username, password pair for authentication
```

Comment:
Looks like use of usernames and passwords has an undesired effect of causing tests to fail when executed in parallel? Should we even document that both can be used in that case? Perhaps we need a check in the check function to verify that credentials are not used when parallel > 1 ?